ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -469,6 +473,10 @@ async def test_update_script(ofrak_client: TestClient, hello_world_elf):
         "",
     ]
 
+    expected_str = normalize(expected_list)
```

Comment:
Does normalize need to be run against the expected output?

Reply:
@whyitfor Running normalize on the expected output (rather than simply testing against a pre-normalized string) allows for copy+pasting the response created from running the steps manually rather than hand-writing them in. I figured this would help reduce future work if the test scripts are changed outside of the normalized areas for any reason.